app/src/main/java/com/okta/appauth/android/example/UserInfoActivity.java

```diff
@@ -179,6 +181,25 @@ private void signOut() {
 
     }
 
+    private void clearData() {
+        Tokens tokens = mOktaAppAuth.getTokens();
```

Comment:
We have a couple of discussions on revocation procedure on iOS and we came up with this design where you have to do revocation manually, but I think we can have an additional revocation method that will do follow the logic you have described so SDK user can choose.

Reply:
Separating the two is fine, this just seems like a lot of code and the OktaAppAuth instance has all of the context (`isUserLoggedIn()`, `getTokens()` etc)

Reply:
That is true, but it could be situations where you need to invalidate only access token but keep current refresh token in use. So I think we can keep both and let users decide. Also, I have changed our example apps for a simpler version where you don't have to provide a specific token.

Reply:
can you explain that use case a bit more?